AppLovinMAXDemo/Plugins/AppLovinMAX/Source/AppLovinMAX/AppLovinMAX_UPL_Android.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--AppLovin plugin additions-->
+<root xmlns:android="http://schemas.android.com/apk/res/android">
+  <!-- init section is always evaluated once per architecture -->
+  <init>
+    <log text="AppLovin Android SDK init"/>
+
+    <!-- check if using Gradle -->
+    <setBoolFromProperty result="bGradle" ini="Engine" section="/Script/AndroidRuntimeSettings.AndroidRuntimeSettings" property="bEnableGradle" default="false"/>
+
+    <!-- get packaging for Applovin from ini and reset it if architecture not supported -->
+    <setBoolFromProperty result="bEnabled" ini="Engine" section="AppLovin" property="bEnabled" default="true"/>
+    <setStringFromProperty result="ClientId" ini="Engine" section="AppLovin" property="ClientId" default="0"/>
+
+    <dumpvars/>
+  </init>
+
+  <!-- optional files or directories to copy or delete from Intermediate/Android/APK after ndk-build -->
+  <resourceCopies>
+    <copyFile src="$S(PluginDir)/../ThirdParty/Android/applovin-max-unreal-plugin.aar" dst="$S(BuildDir)/libs/applovin-max-unreal-plugin.aar" />
+    <copyFile src="$S(PluginDir)/../ThirdParty/Android/applovin-sdk-10.3.0.aar" dst="$S(BuildDir)/libs/applovin-sdk-10.3.0.aar" />
+  </resourceCopies>
+
+  <!-- optional updates applied to AndroidManifest.xml -->
+  <androidManifestUpdates>
+    <if condition="bEnabled">
+      <true>
+        <addPermission android:name="android.permission.INTERNET" />
+        <addPermission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
This is for... InMobi / Tapjoy?

Reply:
I took this from the `AndroidManifest.xml` for the Android SDK, but I admit I'm not sure if that's correct/necessary. If it's not typically needed for integration, then I can check about removing it.